locallibrary/settings.py

```diff
@@ -132,6 +136,9 @@
 
 STATIC_URL = '/static/'
 
+# The absolute path to the directory where collectstatic will collect static files for deployment.
+STATIC_ROOT = BASE_DIR / 'staticfiles'
```

Comment:
đồng nhất folder với STATIC_URL
```suggestion
STATIC_ROOT = BASE_DIR / 'static'
```

Reply:
em sửa r ạ